sys/kern/kern_proc.c

```diff
@@ -2803,44 +2835,67 @@ kern_proc_vmmap_out(struct proc *p, struct sbuf *sb, ssize_t maxlen, int flags)
 			kve->kve_type = KVME_TYPE_NONE;
 			kve->kve_ref_count = 0;
 			kve->kve_shadow_count = 0;
+
+			/* Anonymous memory is presumed able to have tags */
+			kve->kve_protection |= KVME_PROT_LOADTAGS
+					       | KVME_PROT_STORETAGS;
+
+			kve->kve_max_protection |= KVME_PROT_LOADTAGS
+						   | KVME_PROT_STORETAGS;
```

Comment:
I don't think this case is correct any more? ISTR going through and making LOADTAGS/STORETAGS properly inferred everywhere from the object, so if there's no object then you don't get them?

Reply:
Oh indeed.  That's much nicer than I remember it being when I wrote this code.